### ISSUE TITLE
[BUG] "Failed to open file \n Do you want to keep the window open?"

### ISSUE THREAD

**Issue by mlv-** [cboxdoerfer/fsearch#365]
Same as [issue 300](https://github.com/cboxdoerfer/fsearch/issues/300), "stuck on the popup "Failed to open file \n Do you want to keep the window open?" if I press yes or no, nothing happens."

Had to kill the app with xkill after trying to open a folder with an outdated path.

0.13 flatpak version, installed from Pop Store on Pop OS 22.04.

```
System Locale: LANG=en_US.UTF-8
                  LC_NUMERIC=en_CA.UTF-8
                  LC_TIME=en_CA.UTF-8
                  LC_MONETARY=en_CA.UTF-8
                  LC_PAPER=en_CA.UTF-8
                  LC_NAME=en_CA.UTF-8
                  LC_ADDRESS=en_CA.UTF-8
                  LC_TELEPHONE=en_CA.UTF-8
                  LC_MEASUREMENT=en_CA.UTF-8
                  LC_IDENTIFICATION=en_CA.UTF-8
       VC Keymap: us
      X11 Layout: us
       X11 Model: pc105
     X11 Variant: euro
```

**Comment by cboxdoerfer** [comment]
I can confirm the issue, but it seems only to happen when I attempt to open the file with the keyboard (`<Enter>`). Also you should be able to dismiss the dialog with `<Esc>`, so you don't have to kill the app until the patch is ready.

**Comment by cboxdoerfer** [comment]
The bug should be fixed with the 0.1.4 release. If it's still present, let me know and we'll reopen this issue.
